setup.py

```diff
@@ -69,10 +71,10 @@
     'arrow>=0.13.0',
```

Comment:
Do I recall correctly that we are pinning dependencies like this temporarily until we can rely on invenio>=3.2,<4.0 directly because we need specific packages at specific versions?

Another wacky idea: nothing prevents us from passing Pipfile.lock in cookiecutter, right? And then use that `Pipfile.lock`. t won't be as up-to-date necessarily, but it will freeze and only use known working dependencies so we won't get into problem of new versions breaking.

But for now, this is fine.

Reply:
> Do I recall correctly that we are pinning dependencies like this temporarily until we can rely on invenio>=3.2,<4.0 directly because we need specific packages at specific versions?

I was actually thinking about this. I think these deps we are pinning might only be needed for tests?

>Another wacky idea: nothing prevents us from passing Pipfile.lock in cookiecutter, right? And then use that Pipfile.lock. It won't be as up-to-date necessarily, but it will freeze and only use known working dependencies so we won't get into problem of new versions breaking.

The main reason we cannot do that is because we would need to "parametrize" the lock file and matain like 6 lock files depending on ES and DB version/technology. Which is kind of hard due to hashes and so on.